site/content/contribute/getting-started/contribution-checklist.md

```diff
@@ -24,7 +24,7 @@ Follow this checklist for submitting a pull request (PR):
 
     5.1 In the webapp repository run `make i18n-extract` to generate the new/updated strings.
 6. The PR is submitted against the Mattermost `master` branch from your fork.
-7. The PR title begins with the Jira or GitHub Ticket ID (e.g. `MM-394` or `GH-394`) and summary template is filled out.
+7. The PR title begins with the Jira or GitHub Ticket ID (e.g. `[MM-394]` or `[GH-394]`) and summary template is filled out.
```

Comment:
Using braces here is the most common pattern and I updated the docs to reflect that.